src/DynamoCoreWpf/Views/Preview/Watch3DView.xaml.cs

```diff
@@ -216,8 +218,12 @@ private void ThumbResizeThumbOnDragDeltaHandler(object sender, DragDeltaEventArg
 
         private void CompositionTargetRenderingHandler(object sender, EventArgs e)
         {
-            ViewModel.UpdateNearClipPlane();
-            ViewModel.ComputeFrameUpdate();
+            if (!View.Camera.Position.Equals(prevCamera) && !View.IsMouseCaptured )
+            {
+                ViewModel.UpdateNearClipPlane();
+                ViewModel.ComputeFrameUpdate();   
+            }
+            prevCamera = watch_view.Camera.Position;          
         }
```

Comment:
Limit the call to UpdateNearClipPlane() by checking for camera changes and user interaction on the view